### DISCUSSION TITLE
Rnote console commands

### DISCUSSION THREAD

**Discussion by Fra0815** [flxzt/rnote#1323] [Q&A]
So far I haven't found any way to create and open a note file in a specific folder via cmd, I can only open existing files.  I couldn't find any documentation: the '-help' command shows only itself.
<img width="455" alt="image" src="https://github.com/user-attachments/assets/d1b070df-efa6-4eaa-8303-39e59b4f4e85" />

**Comment by Doublonmousse** [comment]
See https://github.com/flxzt/rnote/blob/main/BUILDING.md#configure to activate building the cli.

It is called with `rnote-cli` (see https://github.com/flxzt/rnote/pull/751#issue-1810868784 for examples)

**Reply by Fra0815** [reply]
Ok, thanks for the references.
